source/Templates/How to template.md

```diff
@@ -0,0 +1,46 @@
+# Background
+
+Sashimi uses templates distributed via `dotnet new` command to help developers getting started.
+
+
+
+# Using templates
+
+### Installing latest templates
```

Comment:
I'm not clear when I should call `Installing latest templates` vs `Installing local templates for testing`

Also, can you add an example call to `dotnet new sashimi-create-step -n Blah` please?

Reply:
I'll add to this file